tests/config_file_meta/CONUS/nwm_ana/hydro.namelist

```diff
@@ -18,24 +18,24 @@
     hydrotbl_f = './NWM/DOMAIN/hydro2dtbl_FullRouting_NWMv2.0.nc'
     igrid = 1
     io_config_outputs = 0
-    io_form_outputs = 2
+    io_form_outputs = 4
```

Comment:
I believe we set this to 2 in operations.  Is this just for testing?

Reply:
These are the testing namelists. Not operations. 
This change is required for perfect restarts on channel-only. I guess there are other sneakier ways to do this, but this is the clearest. 
We should eventually have tests for the operational outputs and such, but the current namelists focus on the engineering tests.